pkg/ibmcsidriver/controller.go

```diff
@@ -591,3 +591,9 @@ func (csiCS *CSIControllerServer) ControllerGetVolume(ctx context.Context, req *
 	_ = context.WithValue(ctx, provider.RequestID, requestID)
 	return nil, commonError.GetCSIError(ctxLogger, commonError.MethodUnimplemented, requestID, nil, "ControllerGetVolume")
 }
+
+// ControllerModifyVolume ...
+func (csiCS *CSIControllerServer) ControllerModifyVolume(ctx context.Context, req *csi.ControllerModifyVolumeRequest) (*csi.ControllerModifyVolumeResponse, error) {
```

Comment:
For calling `ControllerModifyVolume` method , do we need `patch` permissions?

Reply:
The patch permission is just for the HonourPVReclaim Policy feature
https://github.ibm.com/alchemy-containers/armada-storage/issues/6528
https://github.com/kubernetes-csi/external-provisioner/pull/1155